# HG changeset patch
# User Rik <rik@octave.org>
# Date 1491602609 25200
#      Fri Apr 07 15:03:29 2017 -0700
# Node ID e284cf4239bff311bd401d100f2f0d62002eee57
# Parent  2a6246cc4a6866a1a7f495aff1b2d1ba7bc0b11d
configure.ac: Print LDFLAGS in summary of config variables.

* configure.ac: Print LDFLAGS in summary of config variables.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -3219,16 +3219,17 @@ Octave is now configured for $canonical_
   Source directory:              $srcdir
   Installation prefix:           $prefix
   C compiler:                    $CC $XTRA_CFLAGS $WARN_CFLAGS $CFLAGS
   C++ compiler:                  $CXX $XTRA_CXXFLAGS $WARN_CXXFLAGS $CXXFLAGS
   Fortran compiler:              $F77 $FFLAGS
   Fortran libraries:             $FLIBS
   Lex libraries:                 $LEXLIB
   LIBS:                          $LIBS
+  LDFLAGS:                       $LDFLAGS
   Extra LDFLAGS:                 $WARN_LDFLAGS
 
   AMD CPPFLAGS:                  $AMD_CPPFLAGS
   AMD LDFLAGS:                   $AMD_LDFLAGS
   AMD libraries:                 $AMD_LIBS
   ARPACK CPPFLAGS:               $ARPACK_CPPFLAGS
   ARPACK LDFLAGS:                $ARPACK_LDFLAGS
   ARPACK libraries:              $ARPACK_LIBS
